services/simcore/docker-compose.yml

```diff
@@ -532,6 +532,34 @@ services:
           memory: 128M
           cpus: '0.1'
 
+  efs-guardian:
+    networks:
+      - monitored
+      - public
```

Comment:
Question: why are these networks needed?

Reply:
No they are not needed, thanks for pointing out. I will remove them.